python/datafusion/expr.py

```diff
@@ -643,3 +666,22 @@ def end(self) -> Expr:
         Any non-matching cases will end in a `null` value.
         """
         return Expr(self.case_builder.end())
+
+
+class SortExpr:
```

Comment:
@timsaucer do we need to redefine this class here or is it okay to continue using expr_internal.SortExpr instead?